src/Files.App/UserControls/SidebarControl.xaml.cs

```diff
@@ -555,35 +491,30 @@ private void NavigationViewItem_RightTapped(object sender, RightTappedRoutedEven
 		{
 			var itemContextMenuFlyout = new CommandBarFlyout { Placement = FlyoutPlacementMode.Full };
 			var sidebarItem = sender as NavigationViewItem;
-			var item = sidebarItem.DataContext as INavigationControlItem;
+			var item = sidebarItem?.DataContext as INavigationControlItem;
```

Comment:
As per the conversation with nullable warnings, should this be reverted?

Reply:
Done.